client/blocks/term-form-dialog/index.jsx

```diff
@@ -212,16 +249,19 @@ class TermFormDialog extends Component {
 				autoFocus={ false }
 				isVisible={ showDialog }
 				buttons={ buttons }
-				onClose={ this.boundCloseDialog }
-				additionalClassNames="term-form-dialog">
-				<FormSectionHeading>{ labels.add_new_item }</FormSectionHeading>
+				onClose={ this.closeDialog }
+				additionalClassNames={ 'term-form-dialog' }>
```

Comment:
I'm not sure why :P I guess I changed it because I thought of adding a new computed classname and changed my mind :)